mrc/configuration/dynamic_configurator.py

```diff
@@ -0,0 +1,40 @@
+import json
+
+from ast import literal_eval
+from kazoo.client import KazooClient
+from kazoo.recipe.watchers import DataWatch
+
+
+class DynamicConfigurator:
```

Comment:
Why add `AbstractConfigurator` if `DynamicConfigurator` doesn't implement it?

Reply:
Because I'm stupid ;)